src/Microsoft.TemplateEngine.Utils/CryptoRandom.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Security.Cryptography;
+
+namespace Microsoft.TemplateEngine.Utils
+{
+    public class CryptoRandom : Random
+    {
+        private RandomNumberGenerator _rng = RandomNumberGenerator.Create();
```

Comment:
`RandomNumberGenerator` implements `IDisposable`. I suggest we implement it for `CryptoRandom` as well and dispose the instance when we are done using it.